.github/workflows/main.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - master
       - 'release/**'
+      - qt6-build
```

Comment:
Was this just for testing or do we intend to keep an official qt6-build on the main repo for the transition period?

Reply:
A tired mistake, will fix.